cli/templates/internal/engines/go_text_engine.go

```diff
@@ -4,12 +4,34 @@ import (
 	"bytes"
 	"fmt"
 	"os"
+	"path/filepath"
 	"text/template"
 )
 
 type GoTextEngine struct {
 }
 
+var commonTemplateFuncs map[string]interface{}
+
+// relativeToCurrentWorkingDir returns a path relative to current working dir.
+// In case of error, fullpath is returned.
+func relativeToCurrentWorkingDir(fullpath string) string {
```

Comment:
Hmm, I don't understand, "How does this change relate to the current commit?"

Reply:
This is required to print more compact path in vshard bootstrapping command.